Sources/swift-doc/Subcommands/Generate.swift

```diff
@@ -72,11 +78,11 @@ extension SwiftDoc {
         for symbol in module.interface.topLevelSymbols.filter(symbolFilter) {
           switch symbol.api {
           case is Class, is Enumeration, is Structure, is Protocol:
-            pages[route(for: symbol)] = TypePage(module: module, symbol: symbol, baseURL: baseURL, includingChildren: symbolFilter)
+            pages[route(for: symbol)] = TypePage(module: module, symbol: symbol, baseURL: baseURL, datesLocale: datesLocale, includingChildren: symbolFilter)
```

Comment:
Please avoid long lines, as with these changes it now requires horizontal scrolling to review even on a wide screen, and makes it even harder on narrow screens. This is applicable to the rest of the long lines in this diff.

```suggestion
            pages[route(for: symbol)] = TypePage(
              module: module,
              symbol: symbol,
              baseURL: baseURL,
              datesLocale: datesLocale,
              includingChildren: symbolFilter
            )
```

Reply:
Would be cool to some swiftformat and swiftlint config to automate this :P
What do you think?
I can help to set it up if you want

Reply:
Thank you, sounds good to me! @mattt WDYT?

Reply:
If we are going to add linting, I would suggest that we should stick to swift-format for linting rather than using swift-lint.  As to setting that up, its pretty trivial (there is an example at https://github.com/compnerd/swift-win32/blob/main/.github/workflows/lint.yml).